document/js-api/index.bs

```diff
@@ -556,22 +515,31 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
 
     * \[[Memory]] : a [=memory address=]
     * \[[BufferObject]] : an {{ArrayBuffer}} whose [=Data Block=] is [=identified with=] the above memory address
+
+Each [=agent=] has an associated [=ordered map=] known as the <dfn>Memory object cache</dfn> mapping [=memory address=]es to {{Memory}} objects.
```

Comment:
Maybe collect the introduction of these caches in a single place in some earlier section?

Reply:
Any idea for where? Currently, all stuff with agents happens at the end. I imagine it'd be less clear if I moved these things to the end, though.

Reply:
There are plenty of forward refs to the "associated store" already, and these are no different. They are part of the same mapping mechanism, so should best be defined in one place. Maybe the section about store interaction could be moved to the beginning?

Reply:
A "store" section makes sense.

Reply:
OK, moved it to a store section, PTAL.